packages/mobile/src/pincode/PincodeSet.test.tsx

```diff
@@ -134,4 +136,36 @@ describe('Pincode', () => {
     await flushMicrotasksQueue()
     expect(getByText('pincodeSet.pinsDontMatch')).toBeDefined()
   })
+
+  it('navigates to the Settings screen after successfully changing PIN', async () => {
+    const { getByTestId, rerender } = render(
+      <Provider store={mockStore}>
+        <PincodeSet {...mockScreenProps} />
+      </Provider>
+    )
+
+    // Create pin
+    mockPin.split('').forEach((number) => fireEvent.press(getByTestId(`digit${number}`)))
+    jest.runAllTimers()
+    await flushMicrotasksQueue()
+    expect(mockScreenProps.navigation.setParams).toBeCalledWith({
+      isVerifying: true,
+      changePin: true,
+    })
+
+    rerender(
+      <Provider store={mockStore}>
+        <PincodeSet
+          {...getMockStackScreenProps(Screens.PincodeSet, { isVerifying: true, changePin: true })}
+        />
+      </Provider>
+    )
+
+    // Verify pin
+    mockPin.split('').forEach((number) => fireEvent.press(getByTestId(`digit${number}`)))
+    jest.runAllTimers()
+    await flushMicrotasksQueue()
+
+    expect(navigateClearingStack).toBeCalledWith(Screens.Settings)
```

Comment:
I think you want to expect `navigate` here not `navigateClearingStack`

Reply:
i was thinking this too but for some reason, when i try to call navigate here it wont show any screens being called, and with navigateClearingStack the VerificationEducationScreen is actually popping up

Reply:
![image](https://user-images.githubusercontent.com/12700801/114918342-d57bef80-9ddb-11eb-999f-f7184a32622f.png)

Reply:
![image](https://user-images.githubusercontent.com/12700801/114918410-e75d9280-9ddb-11eb-8561-adfd5ffb693c.png)